python_modules/dagster/dagster/core/definitions/graph_definition.py

```diff
@@ -460,7 +460,8 @@ def to_job(
         version_strategy: Optional[VersionStrategy] = None,
         op_selection: Optional[List[str]] = None,
         partitions_def: Optional["PartitionsDefinition"] = None,
```

Comment:
<!-- GRAPHITE_METADATA {"trueLocation":{"startLine":455,"endLine":462,"prLink":"https://app.graphite.dev/github/pr/dagster-io/dagster/7498"}} -->
<i class='graphite__hidden'>[Re: lines 455 to 462]<br/></i>

is `resource_defs` the only one of these properties that should be possible to inherit from the repository def? <span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/7498?utm_source=unchanged-line-comment">Graphite</a>.</span>